frontend/src/app/keyresult/key-result-overview/key-result-overview.component.html

```diff
@@ -62,7 +62,19 @@
 
           <ng-template #noMeasures>
             <mat-card-content data-test-marker="emptyLastMeasure">
-              -
+              <p>-</p>
+              <mat-card-actions align="start">
```

Comment:
Ist mir beim vorderen Review nicht aufgefallen: Ist es eine gute Idee, ein `<mat-card-actions>` in ein `<mat-card-content>` zu setzen? Ich denke, die drei MatCard-Titel, -Content und -Action müssen als Geschwister auftreten.
Es tritt aktuell kein Problem auf, aber sauber ist es wahrscheinlich nicht.